### DISCUSSION TITLE
Focus on completion menu immediately after pressing tab

### DISCUSSION THREAD

**Discussion by dgudim** [akinomyoga/ble.sh#364] [Q&A] [answered]
I searched for a config option, but couldn't find one. 
Currently, when I press tab after `cd` for example, a menu with directories opens but the first item is not focused, so I need to press tab again to focus it and start navigating the menu. How can I make the menu navigatable after the fist tab? Thanks

**Comment by akinomyoga** [comment, marked as answer]
It is described around [§7.1.1 and §7.1.2](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A77-Completion#711-widget-complete):

```bash
ble-bind -f TAB menu-complete
ble-bind -f C-i menu-complete
```

You can also set it up through the Readline interface:
```bash
bind '"\C-i": menu-complete'
```

**Reply by dgudim** [reply]
Thank you very much
